File: c++/bmi.cpp



#include <iostream>
using namespace std;
int main(int argc, char **argv)
{
	float waga;
    float bmi;
    float wzrost;
    cout <<"Podaj swoją wage : ";
    cin >> waga;
    cout <<"Podaj swój wzrost: ";
    cin >> wzrost;
    bmi=waga/(wzrost*wzrost);
    if (bmi<18.5)
    { 
    cout <<"niedowaga"; 
}
    if (18.5 <= bmi && bmi <24.9)
    {
    cout <<"norma";
}
    if (25 <= bmi && bmi <30)
    {
    cout << "nadwaga";
}
    if (30 < bmi )
    {
    cout <<"Otyłość";    
}

	return 0;
}

File: python/bmi.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-


def main(args):
    masa=float(input("Podaj swoją wagę: "))
    wzrost=float(input("Podaj swój wzrost:"))
    bmi=masa / (wzrost * wzrost)
    if bmi < 18.5:
        print("niedowaga")
    elif bmi < 24.9:
        print("norma")
    elif bmi < 30:
        print("nadwaga")
    elif 30 <= bmi:
        print("otyłość")


    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
